static/js/containers/admin/EditChannelAppearancePage.js

```diff
@@ -142,6 +174,8 @@ const mapStateToProps = (state, ownProps) => {
   }
 }
 
+export { EditChannelAppearancePage }
```

Comment:
Is there a reason to export like this instead of exporting inline with the class def?